pkg/provisioner/templates/containerd.go

```diff
@@ -32,29 +32,54 @@ const containerdTemplate = `
 with_retry 3 10s sudo apt-get update
 install_packages_with_retry ca-certificates curl gnupg -y
 sudo install -m 0755 -d /etc/apt/keyrings
-curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
-sudo chmod a+r /etc/apt/keyrings/docker.gpg
 
-# Add the repository to Apt sources:
-echo \
-  "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
-  "$(. /etc/os-release && echo "$VERSION_CODENAME")" stable" | \
-  sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
-  with_retry 3 10s sudo apt-get update
+# Download containerd 2.0 from GitHub
```

Comment:
Nit: comment refers to a specific version.

Reply:
Done